### PR TITLE
OpenSCAP json tailoring (HMS-3827)

### PR DESCRIPTION
**Pull request by kingsleyzissou**
This pull request includes:
- changes to the blueprints to enable json tailoring
- update the cloudapi to accept json tailoring options

&nbsp;
- [x] adequate testing for the new functionality or fixed issue
- [x] adequate documentation informing people about the change such as
  - [ ] submit a PR for the READMEs [listed here](https://github.com/osbuild/osbuild.github.io/blob/main/readme-list)
  - [x] submit a PR for the [osbuild.org website](https://github.com/osbuild/osbuild.github.io/) repository if this PR changed any behavior not covered by the automatically updated READMEs
  
Depends on: https://github.com/osbuild/images/pull/798

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/osbuild/osbuild-composer
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.5
```
Comment (croissanne) [inline go.mod:5]: I'm not sure if we can do this, won't mock builds fail if 1.22 isn't available?

**Review by croissanne — COMMENTED** [review]
lgtm but one question about the go.mod changes.

File: go.mod (hunk 1 above)
Comment (kingsleyzissou) [reply go.mod:5]: Ah frik. Sorry that was a mistake on my part. I think I accidentally updated go on my system

Comment (kingsleyzissou) [reply go.mod]: Should be fixed now

**Review by croissanne — APPROVED** [review]
thank you!

**Pull request merged**